<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewProjectWidget</class>
 <widget class="QWidget" name="AddNewProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>58</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QGroupBox" name="background_father_widget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>390</width>
           <height>308</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox{
border-radius: 16px;
border:none;
background-color: rgba(255,255,255,220);
}
</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>47</number>
          </property>
          <property name="topMargin">
           <number>47</number>
          </property>
          <property name="rightMargin">
           <number>47</number>
          </property>
          <property name="bottomMargin">
           <number>47</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="icon_bot_button">
              <property name="minimumSize">
               <size>
                <width>77</width>
                <height>77</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>77</width>
                <height>77</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../bot_icons.qrc">
                <normaloff>:/icons/widgets/times_icon/newProject.png</normaloff>:/icons/widgets/times_icon/newProject.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>77</width>
                <height>77</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLineEdit" name="name_bot">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit{
border:none;
color: #373737;
border-radius:8px;
background-color: white;
}
QLineEdit::hover{
border:none;
color: #black;
border-radius:8px;
background-color: 373737;
}</string>
                </property>
                <property name="text">
                 <string>Новый бот</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Time to start</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="token_bot_edit">
            <property name="minimumSize">
             <size>
              <width>307</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
border:1px solid #373737;
color: #373737;
border-radius:8px;
background-color: transparent;
}</string>
            </property>
            <property name="text">
             <string>Token bot's</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="about_bot_edit">
            <property name="minimumSize">
             <size>
              <width>307</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
border:1px solid#373737;
color: #373737;
border-radius:8px;
background-color: transparent;
}</string>
            </property>
            <property name="text">
             <string>About bot's</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="cancel_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
border:none;
border-radius: 8px;
background-color: #FF5F8F;
color:white;
}</string>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="accept_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
border:none;
border-radius: 8px;
background-color: #39B292;
color:white;
}</string>
              </property>
              <property name="text">
               <string>Accept</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="close_button">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: none;
}

QPushButton:hover{
	border:1px solid grey;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../bot_icons.qrc">
           <normaloff>:/icons/widgets/times_icon/closeIcon.png</normaloff>:/icons/widgets/times_icon/closeIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bot_icons.qrc"/>
 </resources>
 <connections/>
</ui>
